2-mandatory/4-people-i-know.js

```diff
@@ -378,14 +378,23 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+friends.map((friend) => {
```

Comment:
`.map()` returns an array, but it isn't being assigned to a variable here. We just want to loop through the array, so using a `.forEach()` makes more sense.